features/0794-did-rotate/README.md

```diff
@@ -19,6 +19,10 @@ This protocol is only applicable to DIDComm v1 - in DIDComm v2 use the more effi
 This mechanism allows a party in a relationship to change the DID in use for the relationship. This may be used to switch DID methods,
```

Comment:
Tweak — “their DID in use” instead of “the DID in use"

Reply:
I've gone with :
>This mechanism allows a party in a relationship to change the DID they use to identify themselves in that relationship.